### PR TITLE
Add sed_check and sed_take ownership tests

### PR DESCRIPTION
**Pull request by aarthi-manivel**
Introduce sed_check and sed_take_ownership tests.

### CODE REVIEW

File: src/autoval_ssd/tests/sed_check/sed_take_ownership.py [hunk 1]
```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python3
+
+# pyre-unsafe
+"""
+Test validates if the Self Encrypting Drive supports OPAL 2.0 spec
+"""
```
Comment (afagaj) [inline src/autoval_ssd/tests/sed_check/sed_take_ownership.py]: Does this comment correctly describe the current test?  

It seems to be cut-and-paste from sed_check.py

File: src/autoval_ssd/tests/sed_check/sed_take_ownership.py [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+from autoval.lib.utils.autoval_errors import ErrorType
+from autoval_ssd.lib.utils.sed_util import SedUtils
+from autoval_ssd.lib.utils.storage.nvme.nvme_drive import OwnershipStatus
+from autoval_ssd.tests.sed_check.sed_check import SedCheck
+
+
+class SedTakeOwnership(SedCheck):
+    """
+    This script is used to validate if the Self Encrypting drive is based
+    on OPAL 2.0 specification.
```
Comment (afagaj) [inline src/autoval_ssd/tests/sed_check/sed_take_ownership.py]: Does this comment correctly describe the current test?

It seems to be cut-and-paste from sed_check.py

File: src/autoval_ssd/tests/sed_check/sed_take_ownership.py (hunk 1 above)
Comment (aarthi-manivel) [reply src/autoval_ssd/tests/sed_check/sed_take_ownership.py]: Modified to address the functionality

File: src/autoval_ssd/tests/sed_check/sed_take_ownership.py (hunk 2 above)
Comment (aarthi-manivel) [reply src/autoval_ssd/tests/sed_check/sed_take_ownership.py]: Modified to address the functionality

**Review by lclarke1 — APPROVED** [review]

**Pull request merged**